zulip_bots/zulip_bots/bots/wikipedia/doc.md

```diff
@@ -0,0 +1,13 @@
+# Wikipedia bot
+
+## Intro

```

Comment:
Don't use short forms in documentation.

Reply:
Perhaps add a screenshot of it running.

Reply:
What do you mean by short forms? I haven't written any documentation before, still new to this haha.

Reply:
The full form would be `Introduction`, but I think in this case @roberthoenig  updated the task to just remove that heading entirely. So you should remove the `## Intro` line.

Reply:
Ok, thanks.